src/helpers/libs/ethers.js

```diff
@@ -721,41 +749,23 @@ const EthersCalls = {
       filter.toBlock = args[1] ? args[1] : 'latest';
       filter.address = args[2] ? args[2] : null;
 
-      return provider.getLogs(filter);
+      const filterId = await provider.send('eth_newFilter', [filter]);
+      return provider.send('eth_getFilterChanges', [filterId]);
     },
     codeSample: (url, ...args) => {
-      return `const ethers = require("ethers");
-// OR import ethers from 'ethers';
-
-const filter = {
-    topics: ${
-      args[3]
-        ? JSON.stringify(
-            args[3].split(',').map((x) => (x === 'null' ? null : x.split('||')))
-          )
-        : '[]'
-    },
-    ${args[0] ? "fromBlock: '" + args[0] + "'" : "fromBlock: 'latest'"},
-    ${args[1] ? "toBlock: '" + args[1] + "'" : "toBlock: 'latest'"},${
-        args[2] ? "\n\taddress: '" + args[2] + "'" : ''
-      }
-};
-
-// HTTP version
-(async () => {
-  const provider = new ethers.providers.JsonRpcProvider('${url}');
-  const logs = await provider.getLogs(filter);
-  console.log(logs);
-})()
-
-
-// WebSocket version
-(async () => {
-  const provider = new ethers.providers.WebSocketProvider('${url}');
-  const logs = await provider.getLogs(filter);
-  console.log(logs);
-})()
-`;
+      const filter = `const filter = {
+  ${args[0] ? "fromBlock: '" + args[0] + "'" : "fromBlock: 'latest'"},
+  ${args[1] ? "toBlock: '" + args[1] + "'" : "toBlock: 'latest'"},
+  ${args[2] ? "address: '" + args[2] + "'" : ''},
```

Comment:
Because otherwise you'd get a syntax error on copy/paste right?
```suggestion
  ${args[2] ? "address: '" + args[2] + "'," : ''}
```

Reply:
fixed